docs/notebooks/index.rst

```diff
@@ -0,0 +1,13 @@
+Examples
```

Comment:
I don't understand why the need for the new files ?

Reply:
Aesthetics. It gives the notebooks a nested menu. If you aren't a fan, I'll remove it.

Reply:
ah ok! I'm good with the nested menu.